app/services/store_participant_eligibility.rb

```diff
@@ -15,6 +15,7 @@ def call
 
     update_and_store_eligibility_values!
 
+    RecordDeclarations::Actions::MakeDeclarationsEligibleForParticipantProfile.call(participant_profile: participant_profile) if changed_to_eligible?
```

Comment:
`refresh_payability!` has a similar check, right? Should the `MakeDeclarationsEligibleForParticipantProfile` job just call `refresh_payability` on the relevant declarations?

Reply:
Refresh payability isn't required any longer. It's removed by a later commit